demo/index.html

```diff
@@ -72,18 +78,15 @@
               "vAxis": {"minValue" : 0, "maxValue": 10},
               "animation": {"duration": "1000"}}'
     cols='[{"label": "Data", "type": "string"},{"label": "Value", "type": "number"}]'
-    rows='[ ["Col1", 5.0],["Col2", 5.0],["Col3", 5.0] ]'>
-  </google-chart>
+    rows='[ ["Col1", 5.0],["Col2", 5.0],["Col3", 5.0] ]'></google-chart>
 
   <script>
     (function() {
-      function getRandomValue() {
-        return Math.random() * 10;
-      }
+      const getRandomValue = () => Math.random() * 10;
```

Comment:
Don't think we should use arrow functions yet. IE 10/11 and Safari < 10 will have issues.